plugins/Live/API.php

```diff
@@ -146,6 +146,14 @@ public function getLastVisitsForVisitor($visitorId, $idSite, $filter_limit = 10,
     }
 
     /**
+     * Proxy for @see getLastVisitsDetails
+     */
+    public function getLastVisitsDetailsWidget($idSite, $period = false, $date = false, $segment = false, $countVisitorsToFetch = false, $minTimestamp = false, $flat = false, $doNotFetchActions = false)
+    {
+        return $this->getLastVisitsDetails($idSite, $period, $date, $segment, $countVisitorsToFetch, $minTimestamp, $flat, $doNotFetchActions);
+    }
```

Comment:
Because each report requires its own API action because of construction of this method: https://github.com/piwik/piwik/blob/2.16.0-rc1/core/Plugin/Report.php#L326

The earlier implementation was a kind of hack I guess.